source/index.html.md

```diff
@@ -1,239 +1,504 @@
+
+1. Reveals the average CO<sub>2</sub> emissions of Cadillac vehicles
+
+2. Reveals the Cadillac vehicle with the least CO<sub>2</sub> emission
+
+3. Reveals the Cadillac vehicles that have emissions data
+
+# Query API
+
+While the Data API allows query behaviour over a single data set, the Query API allow broader control over the Namara data catalog.
+
+This API supports NiQL (the Namara io Query Language), a language for viewing, aggregating, and joining data sets. It's very similar to SQL, so don't worry - no new textbooks.
+
+##<span class="get">GET</span> Meta
+
+>https://api.namara.io/v0/query/meta
+
+This endpoint provides the meta information for querying this Namara instance. It includes the maximum limit per query, the supported formats, and the default format if none is specified. This information may differ, depending on which deployment of Namara you're using. 
+
+*then maybe a chart? req res?*
+
+##<span class="post">POST</span> Query
+
+>https://api.namara.io/v0/query.:format
+
+Dispatches a query and returns the result
+
+### Request
+
+In these requests, the `query` paramter is required, and if the format is `geojson`, the `geojson_feature_key` is required as well - the others are optional.
+
+Parameter | Type | Description
+--------- | ---- | -----------
+format | `string` | Query response format: `csv`, `json`, or `geojson` (default is `json`)
+project_id | `string` | Project ID to use for subscription lookup. If not provided, it is inferred from API Key
+organization_id | `string` | Organization ID to use for subscription lookup. If not provided, it is inferred from API Key
+query (required) | `string` | NiQL query string
+geojson_feature_key (may be required) | `string` | Property name to use as geometry when rendering `geojson`
+
+### Response
+
+>1) EXAMPLE MISSING because showing things in three different formats. FIGURE IT OUT
+
+>2) `{ "error": <Error message> }`
+
+Response | Description
+-------- | -----------
+200: OK | Query executed successfully (*example 1*)
+422: Unprocessable Entity | Something went wrong (*example 2*)
+
+# Query Specification
+
+NiQL is modelled after standard SQL, but only supports read-only `SELECT` statements. A traditional query must at minimum have a `SELECT` and `FROM` clause in order to execute. While the `SELECT` clause can be anything, the `FROM` clause must specify a data set and version to query. 
+
+For example, if we were looking to query a data set with an id of `2a6412c0-b3c9-420e-9487-abd21b664ac1` and the version `en-1`, our minimum query would be:
+
+<code>SELECT *<br/>
+FROM "2a6412c0-b3c9-420e-9487-abd21b664ac1/en-1" AS BramptonHomes</code>
+
+The parser will verify the table name and handle queries that reference it, but using aliases and quotes whenever possible is recommended.
+
+Below is a summary of supported features:
+
+## SELECT Features
+
+
+### COUNT
+
+<code>SELECT COUNT(* | column1)<br/>FROM data-set-id/en-0</code>
```

Comment:
Is there a way to format these as blocks rather than a single-line styling?

Reply:
There is now. #hacked